src/Utils.cpp

```diff
@@ -0,0 +1,22 @@
+#include <chrono>
+#include "Utils.hpp"
+
+namespace gazebo_thruster
+{
+    double sampleNormal(const double& mu, const double& sigma)
```

Comment:
Initializing a RNG is pretty expensive ... get rid of sampleNormal, and initialize the RNG directly as an attribute of the plugin class.

Reply:
Makes sense, thanks. I'll do that.